source/stylesheets/refills/unstyled/_tooltip.scss

```diff
@@ -0,0 +1,36 @@
+.tooltip-item {  
+  display: inline-block;
+  position: relative;
+  text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip {
+    @include position(absolute, null 0 3em 0);
+    @include transition (all 0.2s ease-in-out);
```

Comment:
Actually: no need to [prefix transitions at all anymore](http://caniuse.com/#feat=css-transitions).